packages/babel-core/src/transformation/file/options/config.js

```diff
@@ -183,5 +183,11 @@ module.exports = {
   moduleId: {
     description: "specify a custom name for module ids",
     type: "string"
-  }
+  },
+
+  passPerPresset: {
+    description: "Whether to spawn a traversal pass per a preset. By default all presets are merged.",
+    type: "boolean",
+    default: false
```

Comment:
We could probably get away with enabling this by default in Babel 6. Otherwise we could just leave this disabled and enable it in Babel 7.